src/d3a/models/myco_matcher/__init__.py

```diff
@@ -8,13 +9,17 @@
 
 class MycoMatcher:
     def __init__(self):
-        self.bid_offer_pairs = []
+        ConstSettings.IAASettings.BID_OFFER_MATCH_TYPE = \
+            BidOfferMatchAlgoEnum.CUSTOM.value  # For testing
```

Comment:
Reminder to revert before merging

Reply:
This should be set inside a setup file for an integrationtest.